libyul/AsmData.h

```diff
@@ -70,7 +70,20 @@ struct VariableDeclaration { langutil::SourceLocation location; TypedNameList va
 /// Block that creates a scope (frees declared stack variables)
 struct Block { langutil::SourceLocation location; std::vector<Statement> statements; };
 /// Function definition ("function f(a, b) -> (d, e) { ... }")
-struct FunctionDefinition { langutil::SourceLocation location; YulString name; TypedNameList parameters; TypedNameList returnVariables; Block body; };
+struct FunctionDefinition
+{
+	langutil::SourceLocation location;
+	YulString name;
+	TypedNameList parameters;
+	TypedNameList returnVariables;
+	Block body;
+
+	/// jump "over" to a specific statement
```

Comment:
Please don't extend the AST just for the purpose of code generation. This information should be stored inside EVMCodeTransform.